### ISSUE TITLE
What a pity!

### ISSUE THREAD

**Issue by 375396776** [appliedgocode/what#4]
my go version is 1.12.6,and what is not working well .
`note: module requires Go 1.13`

**Comment by forging2012** [comment]
go get appliedgo.net/what
and use it 
import "appliedgo.net/what"

**Comment by 375396776** [comment]
yeah,i do it ,but `what` must requires Go 1.13,if i want to use this ,must i update my go version to 1.13?

**Comment by christophberger** [comment]
@375396776 , the compiler should attempt to compile the module regardless, according to the [Go 1.12 release notes](https://golang.org/doc/go1.12#modules):

> The go directive in a go.mod file now indicates the version of the language used by the files within that module. It will be set to the current release (go 1.12) if no existing version is present. If the go directive for a module specifies a version newer than the toolchain in use, the go command will attempt to build the packages regardless, and will note the mismatch only if that build fails. 

Did your build fail? Do you get error messages besides the note?

**Comment by 375396776** [comment]
this is the error messages
```
# appliedgo.net/what
../../lbGolandProjects/goProjects/pkg/mod/appliedgo.net/what@v0.1.1/whatis.go:13:13: undefined: log.Writer
note: module requires Go 1.13
```
 i consider that it's because the version `1.12.6` don not have `log.Writer` function ,`whatis.go` used this func,if i run `go build -tags whathappens` ,there is nothing wrong.
so if i want to use `whatis` func to print log ,i must update my golang version to 1.13 or  more new .

**Comment by christophberger** [comment]
Indeed, `log.Writer` is new in Go 1.13. What blocks you from updating your Go toolchain to the latest release?

**Comment by 375396776** [comment]
well,in our team,we want a unified version,now is 1.12.6 .Actually,I do want to update my Go.In your view,if not unified,could the version difference make a terrible thing？

**Comment by christophberger** [comment]
Ok, I see your point. I would recommend keeping the Go version consistent across the team. According to the Go 1 compatibility promise, you should see little or no disruptions when compiling any Go 1.x code with the latest Go version. However, if you are the only one in your team to use the latest Go version, you might unintentionally start using some feature not available in Go 1.12.6 (like I did when using `log.Writer`), and your code will not compile on your team's machines nor on the CI service.

**Comment by 375396776** [comment]
Oh,I got it.Thank you very much:pray:!!!
